core/src/edu/cornell/gdiac/temporary/LevelSelect.java

```diff
@@ -327,28 +277,39 @@ public void draw(){
         // draw each song
         if (selectedLevel>=1){ //check that if there are a song to the left; if so, draw.
             canvas.draw(albumCovers[selectedLevel-1], Color.WHITE, albumCovers[selectedLevel-1].getWidth()/2,
-                    albumCovers[selectedLevel-1].getHeight()/2, albumCoverCoords[0].x,
-                    albumCoverCoords[0].y, 0, cornerScale, cornerScale);
+                    albumCovers[selectedLevel-1].getHeight()/2, albumCoverCoords[selectedLevel-1].x,
+                    albumCoverCoords[selectedLevel-1].y, 0, albumScales[selectedLevel-1]*scale,
+                    albumScales[selectedLevel-1]*scale);
         }
 
         // It an invariant that selectedLevel is a valid index, so we can simply draw.
         canvas.draw(albumCovers[selectedLevel],Color.WHITE,albumCovers[selectedLevel].getWidth()/2,
-                albumCovers[selectedLevel].getHeight()/2,albumCoverCoords[1].x,
-                albumCoverCoords[1].y,0, centerScale, centerScale);
+                albumCovers[selectedLevel].getHeight()/2,albumCoverCoords[selectedLevel].x,
+                albumCoverCoords[selectedLevel].y,0, albumScales[selectedLevel]*scale,
+                albumScales[selectedLevel]*scale);
```

Comment:
I think you have an additional scale multiplication here that is unnecessary because you set the scales in `albumScales`.